modules/cxf/src/main/java/org/mule/module/cxf/WSProxyService.java

```diff
@@ -71,7 +71,7 @@
     private static final String WSDL_PARAM_2 = "&wsdl";
 
     /** This is an internal semaphore, not a property */
-    private boolean lazyInit = true;
+    private boolean lazyInit = false;
```

Comment:
There was a reason to use that flag before, but if now the value is false, then there is no reason to have it